Rakefile

```diff
@@ -133,6 +133,15 @@ begin
 rescue LoadError
 end
 
+desc "Run specs in parallel using turbo_tests"
+task "pspec" do
+  # Try to detect number of CPUs on both Linux and Mac
+  nproc = `(nproc 2> /dev/null) || sysctl -n hw.logicalcpu`.to_i
+
+  # Limit to 6 processes, as higher number results in more time
+  system("bundle", "exec", "turbo_tests", "-n", nproc.clamp(1, 6).to_s)
```

Comment:
We have examples of different ways to run commands in our Rakefile. Is there any difference between them?

```ruby
# Example 1
`npm install`

# Example 2
sh "npx redocly lint openapi.yml"

# Example 3
system("bundle", "exec", "turbo_tests", "-n", nproc.clamp(1, 6).to_s)

```